src/main/eo/org/eolang/math/number.eo

```diff
@@ -288,6 +288,59 @@
           0.0
           1.0
 
+  # @todo #56:90 min. Implement sin using this taylor object.
+  #  You need to code function for coefficient of nth term.
+  #  Accuracy can be decreased because of stack limitation.
+
+  # @todo #56:90 min. Implement cos using this taylor object.
+  #  You need to code function for coefficient of nth term.
+  #  Accuracy can be decreased because of stack limitation.
+
+  # @todo #56:90 min. Implement tg using this taylor object.
+  #  You need to code function for coefficient of nth term.
+  #  Accuracy can be decreased because of stack limitation.
+
+  # @todo #56:90 min. Implement exp using this taylor object.
+  #  You need to code a function for coefficient of nth term.
+  #  Accuracy can be decreased because of stack limitation.
+
+  # Computes series buy the numbers for start and finish
+  # terms and function of coefficient of nth term.
+  [start finish f] > taylor
```

Comment:
@levBagryansky the Taylor series is a special structure in mathematics, and here we just consider the sum of some finite functional sequence. It is also strange that by passing the right boundary as infinity in the answer we get `nan`, isn't that just a Taylor series?
To sum up, may be we should change the name.

Reply:
We don't need to rename it because if series converges performs condition that for every epsilon > 0 exists a finish that from this number difference between our sum and series sum is less than epsilon. So It is always a Taylor series if the series converges. Let's just add this condition.